<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fossil::Internal::OptionsPage</class>
 <widget class="QWidget" name="Fossil::Internal::OptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="commandChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="repoGroupBox">
     <property name="title">
      <string>Local Repositories</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="defaultRepoPathLabel">
        <property name="toolTip">
         <string>Directory to store local repositories by default.</string>
        </property>
        <property name="text">
         <string>Default path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="defaultRepoPathChooser" native="true">
        <property name="toolTip">
         <string>Directory to store local repositories by default.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="userGroupBox">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="defaultUsernameLabel">
        <property name="text">
         <string>Default user:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="defaultUsernameLineEdit">
        <property name="toolTip">
         <string>Existing user to become an author of changes made to the repository.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sslIdentityFileLabel">
        <property name="text">
         <string>SSL/TLS identity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="sslIdentityFilePathChooser" native="true">
        <property name="toolTip">
         <string>SSL/TLS client identity key to use if requested by the server.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="showLogEntriesLabel">
        <property name="text">
         <string>Log count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="logEntriesCount">
        <property name="toolTip">
         <string>The number of recent commit log entries to show. Choose 0 to see all entries.</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="logEntriesWidthLabel">
        <property name="text">
         <string>Log width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="logEntriesWidth">
        <property name="toolTip">
         <string>The width of log entry line (&gt;20). Choose 0 to see a single line per entry.</string>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="timeoutSecondsLabel">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="timeout">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>213</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QCheckBox" name="disableAutosyncCheckBox">
        <property name="toolTip">
         <string>Disable automatic pull prior to commit or update and automatic push after commit or tag or branch creation.</string>
        </property>
        <property name="text">
         <string>Disable auto-sync</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
